### DISCUSSION TITLE
Is it possible to embed pages from Sentry.io to another Website?

### DISCUSSION THREAD

**Discussion by Stephen-Bartel** [getsentry/support#16] [Support] [answered]
Currently, I have a **Sentry.io** account and push errors to the dashboard interface. 

I was wondering if the following is possible: 

 - Take the pages on the existing **Sentry.io** dashboard
 - Embed some of the panels and pages in a page on a custom website
 - Errors and interface would appear the same way on the custom website
 - Can interact with the interface and errors in the same way in terms of viewing sub-tabs and sub-pages

I'm primarily interested in the: 

 - **Projects** tab and navigate to subtabs like **Project Details** and individual errors
 - **Issues** tab and the ability to search and view errors

Thank you.

**Comment by markstory** [comment, marked as answer]
We don't allow sentry.io to be placed within an iframe/frame so your options for embedding are pretty limited right now. It sounds a bit like you'd like to make a dashboard out of the data in your sentry organization. Depending on what specific panels you want you might be able to build a dashboard of those elements.
